capabilities.json

```diff
@@ -82,45 +124,29 @@
       }
     },
     "tooltip": {
-      "displayName": "Tooltip",
-      "displayNameKey": "Visual_Tooltip",
       "properties": {
         "displayUnits": {
-          "displayName": "Display Units",
-          "displayNameKey": "Visual_DisplayUnits",
           "type": {
             "formatting": {
               "labelDisplayUnits": true
             }
           }
         },
         "precision": {
-          "displayName": "Decimal Places",
-          "displayNameKey": "Visual_Precision",
           "type": {
             "numeric": true
           }
         }
       }
     },
     "legend": {
-      "displayName": "Legend",
-      "description": "Display legend options",
-      "descriptionKey": "Visual_Description_Legend",
-      "displayNameKey": "Visual_DisplayLegendOptions",
```

Comment:
Do not remove displayNameKey strings

Reply:
I believe I was told to remove these here: https://github.com/microsoft/powerbi-visuals-heatmap/pull/118#discussion_r1276455872. Which one is correct?

Reply:
@uve there is no need to keep displayNameKey here

Reply:
Resolving since this does not appear to be an issue and will for sure go away once the formatting service is used instead